docs/schemas/Marmoset-Metadata/HMBA Marmoset Population.csv

```diff
@@ -0,0 +1,45 @@
+﻿Proposed BICAN Field Name,BICAN UUID,Label,Variable (Short) Name,Definition,Short Description,Human Readable Label,is_required,Permissible Values (incl Null),Description (Permissible Values),Data Type,Version Number,Version Date,Sub Domain Name,Domain Name,Size,Input Restrictions,Min Value,Max Value,Measurement Type
+Local Donor ID,,Local Donor ID,subject_id,,,,,,,text,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Sex at Birth,,Sex at Birth,sex,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Age Value (Years),,Age Value (Years),age_of_death,,,,,,,numeric,,,General Subject Fields,HMBA_Marmoset_Population,,,,,

```

Comment:
I'm not completely sure if it is clear that "Age Value"  means the same of "age_of_death"

Reply:
Yes, it is now age of death.

Reply:
Should be fixed now.